src/core/bpf/restrict_fs/restrict-fs.bpf.c

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Why is this GPL-2.0-only? We cannot accept sources that are not compatible with LGPL-2.1-or-later, as it means the license of the entire project is affected

Reply:
is this source copied/derived from an existing GPL-2.0-only project?

Reply:
It's not copied/derived.

Originally this was set as GPL-2.0 only because the kernel enforces the "license" section in the ELF binary to be "GPL". However I see in https://github.com/systemd/systemd/blob/v249/src/core/bpf/socket_bind/socket-bind.bpf.c#L1-L7 there's a note explaining that LGPL-2.1-or-later downgrades to GPL so it's fine to add it as LGPL-2.1-or-later in the SPDX header.

IANAL but I guess I can add a similar comment here.

Reply:
Yes, the note is correct, please do the same thing here